templates/build.html

```diff
@@ -5,18 +5,18 @@
         <h2>
           {{.Build.Builder}} - {{.Build.BuildNum}}       
           {{if .Build.Info}}
-            {{if .Build.Info.task_id}}
-              (<a href="https://evergreen.mongodb.com/task/{{.Build.Info.task_id}}"> {{.Build.Info.task_id}}</a>) 
+            {{if .Build.Info.TaskID}}
+              (<a href="https://evergreen.mongodb.com/task/{{.Build.Info.TaskID}}"> {{.Build.Info.TaskID}}</a>) 
             {{end}}
           {{end}}
         </h2>
-      <h3><a href="https://evergreen.mongodb.com/lobster/build/{{StringifyId .Build.Id}}/all">Complete logs for all</a></h3>
+      <h3><a href="https://evergreen.mongodb.com/lobster/build/{{.Build.Id}}/all">Complete logs for all</a></h3>
```

Comment:
Currently the code is written to be agnostic to the type of the build ID. I believe the history there is that they wanted to use the build ID as a shard key for logs so they changed it to be a string. Since there was a mix of types in the db they made the code able to handle either type. Now, though, all the build documents with `ObjectId` ids (and other documents with references to build ids) have TTLed so we no longer need to handle it.